library/PhpShell/Input.php

```diff
@@ -2,6 +2,8 @@
 
 class PhpShell_Input extends PhpShell_Entity
 {
+    private const RFC_VERSION_TRESHOLD = 32;
```

Comment:
I took the liberty to refactor this. I do not understand what is so specific with the first 32 ids. Shouldn't we have a boolean flag for this instead?

Reply:
I understand your confusion. It's a 'magic' treshold that splits the actual php binaries from the 'helpers' (eg. vld) and rfc outputs (both of which are not shown in the main 'output')

Reply:
and yes - removing the magic here is definitely a valid option